src/server/pachyderm_test.go

```diff
@@ -95,7 +95,9 @@ func TestSimplePipeline(t *testing.T) {
 		false,
 	))
 
-	commitset, err := c.BlockCommitset(commit1.ID)
+	commitInfo, err := c.InspectCommit(pipeline, "master", "")
+	require.NoError(t, err)
+	commitset, err := c.BlockCommitset(commitInfo.Commit.ID)
```

Comment:
A little late now, I guess, but do you think it's worth making a convenience wrapper for this pattern?
Not sure how representative our tests are of usage of our API

Reply:
I think there are a few options here that we could discuss as a team - if you just want the blocking behavior then `BlockCommit` is pretty straightforwardly what you want.  If you want the full commitset, then you need to know the ID of the commitset (but I suppose we could allow an optional `pfs.Commit` in the request to resolve to get the commitset ID).  Another thing I've been thinking about is having any RPC which can change the DAG return a commitset ID (like `StartCommit` does) - this would affect things like `CreatePipeline`, `CreateBranch`, and `SquashCommit` so you would already have the commitset ID in the client.